shared/containers/TicketStatus/index.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux';
+import pathOr from 'ramda/src/pathOr';
+import TicketStatus from '../../components/TicketStatus';
+import { getTicketStatusOptions } from '../../utilities/getTicketStatus';
+
+const defaultProps = {
+  talks: [],
```

Comment:
I think we just copied this and left it. Need to actually build default props here